record-editor/src/app/shared/interfaces/workflow-object.ts

```diff
@@ -3,6 +3,7 @@ import { JsonPatch } from './json-patch';
 
 export interface WorkflowObject<T = object> {
   id: number;
+  validation_errors?: Array<ValidationError>;
   _extra_data?: {
     validation_errors?: Array<ValidationError>;
```

Comment:
should this field be removed?

Reply:
field was added because we send the `validation_errors` directly instead of a field in `extra_data` in case of backoffice responses. But we cant remove it from `extra_data` either because we send in like this in case of next responses